mongo.tf

```diff
@@ -31,6 +31,26 @@ resource "aws_vpc_peering_connection_accepter" "peer" {
   auto_accept = true
 }
 
+# Route tables for peering connection
+# TODO figure out how to make this DRY using a loop
+resource "aws_route" "peering-owner-az1" {
```

Comment:
@prashant0085 Do you know how to make use of a loop in Terraform to make this DRY?

Reply:
Take a look this article https://www.terraform.io/docs/configuration/resources.html#for_each-multiple-resource-instances-defined-by-a-map-or-set-of-strings

Reply:
@mannykary Sorry, My notification was enabled for the Pandemics group so I couldn't help with this and also other improvements I mentioned earlier. Today after work(Post 3 pm CST) I will look into this.